rebar.lock

```diff
@@ -22,7 +22,7 @@
   0},
  {<<"aesophia">>,
   {git,"https://github.com/aeternity/aesophia.git",
-      {ref,"8b7e4db4909810d23ebfebebaaaa4190d0fac197"}},
+      {ref,"3271d6fba49fd03499b6b93f18130ff822a9f8e3"}},
```

Comment:
https://github.com/aeternity/aesophia/compare/8b7e4db...3271d6f

Reply:
This includes https://github.com/aeternity/aebytecode/compare/34ae94e...2d599d that is consistent.

Reply:
This cleans up blake2b lib used that is ok.

Reply:
This seems reasonably large.